.gitignore

```diff
@@ -102,3 +102,9 @@ dist
 
```

Comment:
It looks like there's a `jspm_packages` above. Is that a typo, and if so, would you be able to fix it here?